### DISCUSSION TITLE
Compatible with Seurat v5?

### DISCUSSION THREAD

**Discussion by bingy007** [GreenleafLab/ArchR#2049] [General]
Hi ArchR team,

Seurat is releasing version 5 recently and some functions are not with the previous names. Is it possible to update ArchR as well since I think ArchR calls Seurat functions in a few steps?

Best,
B

**Comment by rcorces** [comment]
I have not explicitly tested ArchR with Seurat v5 but one would hope the developers of Seurat would ensure backwards compatibility of function calls
